lib/v2/screens/wallet/components/tokens_cards/interactor/viewmodels/token_balances_state.dart

```diff
@@ -19,6 +19,9 @@ class TokenBalancesState extends Equatable {
   @override
   List<Object?> get props => [pageState, errorMessage, availableTokens];
 
+  TokenBalanceViewModel? balanceViewModelForToken(String tokenModelID) =>
```

Comment:
this will crash if none found, read the `firstWhere` documentation 
You need to pass `else` param

Reply:
Damn - fixing...

Reply:
Fixed